### ISSUE TITLE
gather top words across all top languages

### ISSUE THREAD

**Issue by masterkrang** [masterkrang/kako#8]

**Comment by masterkrang** [comment]
doing this in english since this is the only target language i'm familiar with

**Comment by JoseUgalde** [comment]
Are you familiar with the concept of "Semantic Primes"?
Please read the Wikipedia article on it:
https://en.wikipedia.org/wiki/Semantic_primes
Also, check out the three external links referenced in the article, specially 

```
                The Search for the Shared Semantic Core
```

of All Languages 

Date: Mon, 19 Oct 2015 19:06:12 -0700

**Comment by masterkrang** [comment]
@JoseUgalde i am not familiar but i will do some research today. thanks for the suggestion Jose, why do you recommend it?

**Comment by masterkrang** [comment]
I am gathering top 200 words across 20 or more top langauges of the world to determine which are the most commonly used words. you can follow along or see it here: https://docs.google.com/spreadsheets/d/1vq9y94NbHZ1E8ivediRI8jAp5Tzn6TE6aNWT73hpqDE/edit?ts=5626f55e#gid=0

**Comment by masterkrang** [comment]
@JoseUgalde did some research on NSM Natural Semantic Languages and Semantic Primes, here are some thoughts.

An example of NSM would be to not use a word like "lie" and instead explain the situation drawing from a smaller palette of core words. Instead we could use a sentence like "what a person does when he says something not true because he wants someone to think it true". This approach could be powerful at eliminating useless words, or words used infrequently and then forgotten. 

But, I'd argue against this especially in my gross example above. I think in a lot of situations it just creates more work, i.e. "friction". I think we should try to reduce friction wherever we can, but based on certain rules. For example a word like "serendipity" I'd imagine is not used very often, so we may prefer to explain this phenomena (serendipity) with a sentence composed of already established and more simple to use words. 

So I think we'd need a set of rules when deciding when to create new words or just explain meanings. My first guess would be that we create new words when the word has a high frequency. A word like "lie" probably has a massive frequency compared to "serendipity" so it should be it's own word. My guess is that words like lie are more natural and more useful. Some people may never feel or even understand what "serendipity" is. 

In summary, I propose this:
- always make new words for concepts that are likely (or deemed) part of a natural semantic language
- where it is not clear or obvious that a word isn't not part of a NSM or used in low frequency, do not make a new word
- make sure our language includes all word from natural semantic languages in Version 0.1

More info: https://en.wikipedia.org/wiki/Natural_semantic_metalanguage
